### PR TITLE
css for the filter component

### PR DESCRIPTION
**Pull request by Inzurriaga**
this is the css for the filter component

### CODE REVIEW

File: src/Filter.js [hunk 1]
```diff
@@ -94,9 +97,10 @@ export default class Filter extends Component {
           {
             this.state.areas.map((area) => {
               return (
-                <div>
-                  <input onChange={this.UpdateAreasPick} name="areas" type="checkbox" value={area}/> {area}
-                </div>
+                <label class="container">
```
Comment (kimmichurri) [inline src/Filter.js:100]: I like how you changed these to be semantic html.

File: src/Filter.js [hunk 2]
```diff
@@ -75,16 +76,18 @@ export default class Filter extends Component {
         <h3>Distance From Denver</h3>
         <div className="filter-range-section">
           <p>{this.state.setMax} miles away</p>
+          <img className="hiker-image" src={Hiker} alt="hiker"/>
```
Comment (kimmichurri) [inline src/Filter.js:79]: I think the hiker icon looked really great on the preview you showed us.

Comment (codexguajil) [inline src/Filter.js:79]: Nice that you created a separate span for the checkbox. I was thinking we could make it bigger. :)

File: src/Filter.js (hunk 1 above)
Comment (codexguajil) [inline src/Filter.js:100]: Can't wait to see how this looks together.

**Pull request merged**